Lesson009/HW09.Motorcycle/Motorcycle.cs

```diff
@@ -37,17 +33,10 @@ public Motorcycle(string model, string make, DateTime createdAt, string conditio
 
         public string GetAllInfo()
         {
-            var createdAt = "";
-            createdAt += _createdAt;
+            var allInfo = "";
+            allInfo += $"{"Model:",-20}| {_model}\n" + $"{"Make:",-20}| {_make}\n" + $"{"CreatedAt:",-20}| {_createdAt}\n" + $"{"Condition:",-20}| {_condition}\n" + $"{"Mileage:",-20}| {_mileage}\n" + $"{"MotorcycleType:",-20}| {_motorcycleType}\n" + $"{"ExteriorColor: ",-20}| {_exteriorColor}";
 
```

Comment:
1) Use a constant for a repeating and non-changing value
2) Make a line break for better readability of the code.
var result = "123456789" +
                   "987654321"+
                   "000000000";